controllers/user.js

```diff
@@ -37,6 +37,33 @@ const Users = {
       }
     }).catch(() => res.status(500).send({ success: false, message: 'Unexpected error occured' }));
   },
+
+  updateUser(req, res) {
```

Comment:
This does not account for updating an email? 🤔

Reply:
I wanted to prevent that from happening but I think it makes sense to include it